lib/mutant/integration/minitest.rb

```diff
@@ -0,0 +1,155 @@
+require 'minitest'
+
+# monkey patch for minitest
+module Minitest
+  # Prevent autorun from running tests when the VM closes
+  #
+  # Mutant needs control about the exit status of the VM and
+  # the moment of test execution
+  #
+  # @api private
+  #
+  # @return [nil]
+  def self.autorun
+  end
+
+end # Minitest
```

Comment:
If you want to stick with the primary syntax here you could still cut two lines by instead writing

```ruby
def Minitest.autorun
end
```

I know this syntax is usually frowned upon but it might make sense in a context like this. It also has the benefit then of not pinning YARD documentation to the toplevel `Minitest` namespace.